### PR TITLE
Modified phone numbers and added DRS

### PR DESCRIPTION
**Pull request by c2nelson**
Brian, I coded DRS the same as CRT.  Let me know if I should modify to a subheading.  And of course if I need to fix anything else.  I'd be glad to redo this process for accuracy but also training.  I wasn't able to preview it until now.  
Thanks!

### CODE REVIEW

**Comment by c2nelson** [comment]
I need to modify this again.  Let's talk tomorrow.

File: _includes/footer.html [hunk 1]
```diff
@@ -6,12 +6,15 @@
           {% asset doj-logo.png height="64" alt="" %}
         </div>
         <div class="grid-col-auto">
-          <p class="usa-footer__logo-subheading">
+          <p class="usa-footer__logo-heading">
+            ADA.gov
+          </p>
+          <p class="usa-footer_subheading">
```
Comment (hursey013) [inline _includes/footer.html:12]:
```suggestion
          <p class="usa-footer__logo-subheading">
```

File: _includes/footer.html [hunk 2]
```diff
@@ -6,12 +6,15 @@
...
             U.S. Department of Justice
           </p>
-          <p class="usa-footer__logo-heading">
+          <p class="usa-footer__subheading">
```
Comment (hursey013) [inline _includes/footer.html:15]:
```suggestion
          <p class="usa-footer__logo-subheading">
```

**Review by hursey013 — COMMENTED** [review]
A couple of small tweaks to the class names, otherwise this looks good to go!

**Pull request merged**